ctapipe/io/containers.py

```diff
@@ -153,7 +154,10 @@ class R1CameraContainer(Container):
         "numpy array containing a set of images, one per ADC sample"
         "Shape: (n_channels, n_pixels, n_samples)"
     ))
-
+    gain_selection = Field(None, (
```

Comment:
I'd recommend calling this "gain_channel" or "selected_channel"  or "pixel_channel" (which you use elsewhere)  since the values are channel numbers.

Reply:
Since we mix "gains" and "channels" (which both mean the same thing in this context), we should at least try to be consistent.  Maybe even changing all uses of "channel" or "gain" to  "gain_channel" everywhere

Reply:
technically even "gain" is confusing since we use it to refer to DC/PEs as well as flatfielding coefficients as well (which are gain corrections)... the right term is maybe "amplifier channel", since we have "low-gain" and "high-gain" amplifiers.

Reply:
Would people more immediately recognise the meaning of "gain channel" rather than "amplifier channel"?

Reply:
I guess so. 

In Developing FACT-Tools we had a period where we wanted to make every term most `right`, which often meant that it was far away from expectations. In the end, that did more harm than good I would think.

So, we have to compromise between exactness and principal of least surprise.

In this case, I think `gain_channel` and `selected_gain_channel` are a good compromise.

Reply:
Ok, agreed to keep it `gain_channel` and `selected_gain_channel`.